packages/cli-platform-ios/src/commands/buildIOS/index.ts

```diff
@@ -228,6 +228,10 @@ export const iosBuildOptions = [
     description:
       'Explicitly set device to use by name.  The value is not required if you have a single device connected.',
   },
+  {
+    name: '--destination <string>',
+    description: 'Explicitly extend distination e.g. "arch=x86_64"',
+  },
```

Comment:
Please add this flag to the [docs](https://github.com/react-native-community/cli/blob/main/packages/cli-platform-ios/README.md).

Reply:
Added to docs

Reply:
What version of react-native and the cli is this option available from? Doesn't seem to work in 0.70.14 with cli version 9.2.1

Reply:
hey @domthomhive, it's available since 0.72 (CLI version: 11.x).